x/tokenfactory/keeper/grpc_query.go

```diff
@@ -38,3 +39,47 @@ func (q Querier) Params(
 	}
 	return &types.QueryParamsResponse{Params: params}, nil
 }
+
+// Denoms: Returns all registered denoms for a given creator.
+func (q Querier) Denoms(
```

Comment:
I dont see why we need this and the other one, they are very similar

Reply:
It's the DenomsFromCreator query